specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,1067 @@
+        "operationId": "Features_GetItems",
+        "x-ms-examples": {
+          "GetFeatures": {
+            "$ref": "./examples/features/Features_GetItems.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/LimitFeatures"
+          },
+          {
+            "$ref": "#/parameters/Bbox"
+          },
+          {
+            "$ref": "#/parameters/Filter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/ExtendedGeoJsonFeatureCollection"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      },
+      "post": {
+        "description": "The Post Feature API creates a feature with the provided attributes in the given collection.\n\nThe Features API is a part of [Creator](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps), which makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset/create). Datasets consist of multiple feature collections, which are a collection of features of a similar type based on a common schema referred to as ontology. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).",
```

Comment:
Hmm I'm a bit conflicted here because GET, PUT, and PATCH API descriptions all start with the http verb, so using POST here remains consistent across. What are your thoughts on how to keep it consistent?

Reply:
I don't think the consistency was intentional. If someone is reading just this description I don't think it will make sense. I think it can be either "This Post Feature API creates " or "The Create Feature API creates...".

Reply:
I've updated it to "This Post Feature API creates..."